apollo-gradle-plugin-incubating/src/test/kotlin/com/apollographql/apollo/gradle/test/UpToDateTests.kt

```diff
@@ -72,6 +72,27 @@ class UpToDateTests {
     File(dir, "build.gradle").writeText(text.replace(apolloBlock, ""))
   }
 
+  fun `setting singularizeTypes in the build script re-generates the CustomType class`(dir: File) {
```

Comment:
Is this test working?

Reply:
It's removed now, per @martinbonnin's comment